extensions/bokeh_magic.py

```diff
@@ -27,6 +31,11 @@
 class BokehMagics(Magics):
     """Magic to embed Bokeh into the IPython notebook."""
 
+    if old_bokeh:
+        raise RuntimeError("We are deprecating the Bokeh magic. "
```

Comment:
Maybe:
```
%bokeh magic is deprecated, and will not work in versions > 0.7.1
```
Also, `DeprecationWarning`  instead of `RuntimeError` ? I think the notebook is more chatty about showing warnings/errors by default?